packages/extensions/openapi-to-typespec/src/utils/values.ts

```diff
@@ -19,6 +19,16 @@ export function getDefaultValue(type: string, schema: Schema) {
         return `${type}.\`${choice.language.default.name}\``;
       }
     }
+  } else if (isDurationSchema(schema)) {
+    // TODO: need to add back default value when TypeSpec supports
+    return undefined;
```

Comment:
Is this the limitation for typespec that does not allow the default value for duration?

Reply:
Yes. See: https://github.com/Azure/autorest/issues/4875